h/templates/5xx.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <title>Server Error</title>
+      <style>
```

Comment:
Is it a deliberate choice to put CSS inline in the file rather than use our site stylesheet?